spec/mailers/user_activator_spec.rb

```diff
@@ -3,12 +3,11 @@
 RSpec.describe UserActivatorMailer, type: :mailer do
   it "sends an email" do
     # Create the email and store it for further assertions
-    user = User.create(name: 'Megan', email: 'kewlkid@hotmail.com', password: 'gIrLsRuLeBoYsDrOOl')
+    user = User.create(name: 'Megan', email: 'kewlkid@hotmail.com', password: 'gIrLsRuLeBoYsDrOOl', apikey: ENV['USER_KEY'])
```

Comment:
We can switch this over to using the factory I created if we add an API key parameter there.